packages/common/src/search/_internal/hubSearchChannels.ts

```diff
@@ -142,24 +142,25 @@ export const toHubSearchResult = async (
       },
       includes: {
         // when 'include' requests 'groups' enrichment, get group details
-        groups: options.include?.includes("groups")
-          ? await Promise.all(
-              channel.groups.map(async (groupId) => {
-                let group;
-                try {
-                  group = await getGroup(groupId, options.requestOptions);
-                } catch (e) {
-                  group = null;
-                  /* tslint:disable-next-line: no-console */
-                  console.warn(
-                    `Cannot fetch group enhancement for id = ${groupId}`,
-                    e
-                  );
-                }
-                return group;
-              })
-            )
-          : [],
+        groups:
+          options.include?.includes("groups") && channel.groups
```

Comment:
Is this a valid use case? I see some channel records on QA that have `groups: null` that appear to have been created by an `E2E` user. I'm curious if those records were created intentionally or by mistake or if they somehow got into a weird state.. I've not run into this before and suspect this would cause a lot of other issues elsewhere. Those same records also have `access: null`, which I find equally alarming and could be problematic too.

Reply:
I noticed these were created via e2e's.  We should check with backend.  Here's some examples:

<img width="367" alt="Screenshot 2023-11-20 at 3 15 36 PM" src="https://github.com/Esri/hub.js/assets/3883995/5de21658-8261-4209-bc63-8668640800b4">